metriq_gym/process.py

```diff
@@ -38,6 +43,7 @@ def poll_job_results(jobs_file: str, job_type: BenchJobType) -> list[BenchJobRes
     
     with open(jobs_file, "r") as file:
         lines = file.readlines()
+        logging.info(f"{len(lines)} job(s) dispatched.")
```

Comment:
Think this is fine, but `logging` suggests using `%` over `f`-strings for logging statements due to performance issues:
https://stackoverflow.com/questions/54367975/python-3-7-logging-f-strings-vs

Reply:
TIL. Fixed in [53b7ba1](https://github.com/unitaryfund/metriq-gym/pull/14/commits/53b7ba1335df68105a67539e1d53a5c376d89bdc)